lms/static/js/views/fields.js

```diff
@@ -74,24 +99,28 @@
             },
 
             title: function (text) {
-                return this.$('.u-field-title').html(text);
+                return this.$('.u-field-title').text(text);
             },
 
-            getMessage: function(message_status) {
+            getMessageHTML: function(message_status) {
                 if ((message_status + 'Message') in this) {
                     return this[message_status + 'Message'].call(this);
                 } else if (this.showMessages) {
-                    return this.indicators[message_status] + this.messages[message_status];
+                    return HtmlUtils.joinHtml(this.indicators[message_status], this.messages[message_status]);
                 }
                 return this.indicators[message_status];
             },
 
+            getMessage: function(message) {
```

Comment:
I made `getMessage()` to return the string version of `getMessageHtml()` in case other functions called it since `getMessageHtml()` returns a HtmlSnippet now. It was also to help with methods like `showSuccessMessage()` with the `===`. I can put it back to one `getMessage()` fn. 

Do you have any suggestions for the `showSuccessMessage()` issue with the `===`?